src/transport.ts

```diff
@@ -29,6 +30,12 @@ export interface Options {
   token?: string;
   /** A logger with the same interface like console. */
   logger?: Logger;
+  /** Repository owner */
+  owner?: string;
+  /** Repository name */
+  repo?: string;
+  /** Download directory */
+  downloadDirectory?: string;
```

Comment:
These options certainly do not belong in here. You could re-export another `Options` interface from the client like this:

```ts
import { Options as TransportOptions } from './transport';

export interface Options extends TransportOptions {
  /** Repository owner. */
  owner?: string;
  /** Repository name. */
  repo?: string;
  /** Default download directory. */
  downloadDirectory?: string;
}
```

Reply:
👍 cool stuff